Casamari/MNC010.xml

```diff
@@ -33,49 +36,143 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                     <locus target="#34r"/>
+                     <title type="complete" ref="LIT2993RepCh274">Hymn to God</title>
+                     <textLang mainLang="gez"/>
+                     <note>See the transcription in <ref target="#HymnGod1"/>.</note>
+                  </msItem>
+
+                  <msItem xml:id="ms_i5">
+                     <locus from="34r" to="34v"/>
+                     <title type="complete" ref="LIT6232HymnGod">Hymn to God</title>
+                     <textLang mainLang="gez"/>
+                     <note>See the transcription in <ref target="#HymnGod2"/>.</note>
+                  </msItem>
+
+                  <msItem xml:id="ms_i6">
+                     <locus from="34v" to="35r"/>
+                     <title type="complete" ref="LIT2988RepCh269">Hymn to God</title>
+                     <textLang mainLang="gez"/>
+                     <note>See the transcription in <ref target="#HymnGod3"/>.</note>
+                  </msItem>
+
+                  <msItem xml:id="ms_i7">
+                     <locus from="35r" to="35v"/>
+                     <title type="complete" ref="LIT3002RepCh282">Hymn to God</title>
+                     <textLang mainLang="gez"/>
+                     <note>See the transcription in <ref target="#HymnGod4"/>.</note>
+                  </msItem>
+
+                  <msItem xml:id="ms_i8">
+                     <locus from="35v" to="38v"/>
+                     <title type="complete" ref="LIT4963Naahabeya"/>
                      <textLang mainLang="gez"/>
+                     <incipit xml:lang="gez">
+                        <locus target="#35v"/>
+                        <hi rend="rubric">ነዓ፡ ኀቤየ፡ ኦእግዚእየ፡ ኢየሱስ፡ ክርስቶስ፡ ወልደ፡ እግዚአ</hi><del
+                           rend="erasure" cause="superfluous">እግዚአ</del>ብሔር፡ ሕያው። ወወልደ፡ <hi
+                           rend="rubric">ማርያም፡</hi> ሥግው። መ<del rend="erasure" cause="superfluous"
+                           >ለ</del>ብረቀ፡ <hi rend="rubric">መለኮት፡</hi> መደንግፅ። ፍሕመ፡ <hi rend="rubric"
+                           >መለኮት፡</hi> ብቁጽ። ሐፀ፡ <hi rend="rubric">መለኮት፡</hi> ርውፅ።</incipit>
+                     <explicit xml:lang="gez">
+                        <locus target="#38v"/>እጼለል። ሳዶር፡ አላዶር፡ ዳናት፡ አዴራ፡ ሮዳስ፡ በዝንቱ፡ አስማተ፡ ፭ ቅንዋተ፡
+                        መስቀሉ፡ ለእግዚእነ፡ <hi rend="rubric">ኢየሱስ፡ ክርስቶስ፡</hi> ዘቦቱ፡ ለስሐ፡ ሕምዙ፡ ለሞት።
+                        ወተቀጥቀጠ፡ ኃይሎሙ፡ ለሰይጣናት። ከማሁ፡ ይልሳሕ፡ ሕምዞሙ፡ ለአጋንንት፡ ወይትቀጥቀጥ፡ ኃይሎሙ፡ ለሰይጣናት፡ ከመ፡
+                        ኢይቅረቡ፡ ኀበ፡ ነፍሳ፡ ወሥጋሃ፡ ለዓመትከ፡ <persName ref="PRS13232EhetaMar"><hi
+                              rend="rubric">እኅተ፡ ማርያም።</hi></persName> ለዓለመ፡ ዓለም።</explicit>
                   </msItem>
+
+                  <msItem xml:id="ms_i9">
+                     <locus target="#39r"/>
+                     <title ref="LIT2715John">Excerpt from the Gospel of John</title>
+                     <note>The excerpt comprises <ref type="work" corresp="LIT2715John.1.1-5">John
```

Comment:
Silly question: is this way to indicate biblical excerpts (in particular the section `.1.1-5` after the ID) supported in the guidelines or by someone else's use? I've adopted a different system.  
Another point: we need to decide in a future meeting how to standardize references to biblical quotations: e.g., in the present case, John 1,1-1,5 or Jn 1:1-1:5 etc I'd opt for the book abbreviation (Jn instead of John).